From 0e4b4f547d60228629760d26c9eccf305bad2ba6 Mon Sep 17 00:00:00 2001
Message-ID: <0e4b4f547d60228629760d26c9eccf305bad2ba6.1700222319.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1700222319.git.sharlatanus@gmail.com>
References: <cover.1700222319.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 Nov 2023 20:13:33 +0000
Subject: [PATCH 27/32] gnu: python-pytest-socket: Update to 0.6.0.

* gnu/packages/python-check.scm (python-pytest-socket): Update to 0.6.0.
[source]: File name changed in PyPI repository.

Change-Id: I3dd984c6f752377b43decd201d184fc2da8daf60
---
 gnu/packages/python-check.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index ea74472270..e7de83539d 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -422,13 +422,13 @@ (define-public python-vcrpy
 (define-public python-pytest-socket
   (package
     (name "python-pytest-socket")
-    (version "0.5.1")
+    (version "0.6.0")
     (source (origin
               (method url-fetch)
-              (uri (pypi-uri "pytest-socket" version))
+              (uri (pypi-uri "pytest_socket" version))
               (sha256
                (base32
-                "1dkr86nxkxc0ka3rdnpmk335m8gl1zh1sy8i7w4w1jsidbf82jvw"))))
+                "0f3irpd8lwmm3pb1fqnvwg82kpkhsnzsmw8jg7yd85c349kisg1n"))))
     (build-system python-build-system)
     (arguments
      ;; FIXME: Tests fail a lot, probably requiring Internet access.
-- 
2.41.0

